specs/brew/install_all_dependencies.yaml

```diff
@@ -0,0 +1,10 @@
+---
+name: Install all dependencies from Brewfile
+command: brew bundle [install]
```

Comment:
should this be `brew bundle install` instead?

Reply:
Yes, updated it. Please check now.